src/cclib/parser/orcaparser.py

```diff
@@ -151,7 +151,7 @@ def extract(self, inputfile, line):
             if not hasattr(self, "scftargets"):
                 self.scftargets = []
 
-            energy = self.scfvalues[-1][-1][0]
```

Comment:
At first I thought I made a mistake here, but now I'm not so sure. I have a different output where the first geometry step is unconverged:

```
...
123  -2921.2788160254  -0.000000336370 0.00012032  0.00000135  0.0000431 0.0000
124  -2921.2788164185  -0.000000393072 0.00012020  0.00000135  0.0000419 0.0000

*****************************************************
*                      ERROR                        *
*        SCF NOT CONVERGED AFTER 125 CYCLES         *
*****************************************************
...
---------------
SCF CONVERGENCE
---------------

Last Energy change         ...   -3.9307e-07  Tolerance :   1.0000e-08
Last MAX-Density change    ...    1.2020e-04  Tolerance :   1.0000e-07
Last RMS-Density change    ...    1.3465e-06  Tolerance :   5.0000e-09
Last DIIS Error            ...    4.1883e-05  Tolerance :   5.0000e-07
...

FINAL SINGLE POINT ENERGY     -2921.277851575335
...
122  -2921.3318360174  -0.000000001792 0.00001354  0.00000010  0.0000050 0.0000
123  -2921.3318359899   0.000000027559 0.00001309  0.00000010  0.0000049 0.0000
                 **** Energy Check signals convergence ****

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER 124 CYCLES          *
               *****************************************************
...
Total Energy       :        -2921.33098298 Eh          -79493.45742 eV
  Last Energy change         ...   -1.4893e-09  Tolerance :   1.0000e-08
  Last MAX-Density change    ...    1.2671e-05  Tolerance :   1.0000e-07
...
FINAL SINGLE POINT ENERGY     -2921.330982975234
...
```
With the current version of the parser (pre-pull request),
```
scfvalues[0][-1] == array([ -3.93070000e-07,   1.20200000e-04,   1.34650000e-06])
scfvalues[1][-1] == array([ -1.48930000e-09,   1.26710000e-05,   1.00000000e-07])
scftargets[0] == array([  1.00000000e-08,   1.00000000e-07,   5.00000000e-09])
scftargets[1] == array([  1.00000000e-08,   1.00000000e-07,   5.00000000e-09])
scfenergies[0] == -3.9307200000000001e-07
scfenergies[1] == -79493.457420000006
```
With the version of the parser from this pull request,
```
scfvalues[0][-1] == array([ -3.93070000e-07,   1.20200000e-04,   1.34650000e-06])
scfvalues[1][-1] == array([ -1.48930000e-09,   1.26710000e-05,   1.00000000e-07])
scftargets[0] == array([  1.00000000e-08,   1.00000000e-07,   5.00000000e-09])
scftargets[1] == array([  1.00000000e-08,   1.00000000e-07,   5.00000000e-09])
scfenergies[0] == -1.06960335443736e-05
scfenergies[1] == -79493.462236363775
```
So, it looks like there's no conversion for `scfvalues/scftargets` from hartree. I'm not sure what the other parsers do, since they all use different convergence criteria, and sometimes not delta_E at all. So, I think we should be self-consistent and not convert this line to eV unless *all* of the energies in `scfvalues/scftargets` are converted to eV. Is that a bug or an oversight?